### ISSUE TITLE
vulnerability scan report 

### ISSUE THREAD

**Issue by puppy61** [aquasecurity/trivy#3147]
## Output of `trivy -v`:
```
Version: 0.29.2
Vulnerability DB:
  Version: 2
  UpdatedAt: 2022-11-08 00:13:56.935988406 +0000 UTC
  NextUpdate: 2022-11-08 06:13:56.935987906 +0000 UTC
  DownloadedAt: 2022-11-08 02:10:40.572864529 +0000 UTC
```
![image](https://user-images.githubusercontent.com/112548224/200484349-66511934-f7f9-4f8b-8574-6499d352ae59.png)

I use trivy for harbor image vulnerability  scan.  Accordding to the  cve description，CVE-2022-3358 only affects ’OpenSSL versions 3.0.0 to 3.0.5‘， why image  with openssl-1.0.1t-1+deb8u7  be reported  existing high risk vulnerabilities.

Thanks.

**Comment by DmitriyLewen** [comment]
Hello @puppy61 
Thanks for your report!

For Debian packages Trivy uses Debian database(https://aquasecurity.github.io/trivy/v0.34/docs/vulnerability/detection/data-source/#data-source-selection)

https://security-tracker.debian.org/tracker/CVE-2022-3358:
For `buster` fixed version == `1.1.1n-0+deb10u3`
For `bullseye` fixed version == `1.1.1n-0+deb11u3`

That is why Trivy finds this CVE.

Regards, Dmitriy

**Comment by knqyf263** [comment]
Duplicate of https://github.com/aquasecurity/trivy/issues/3194
